connector/connectionProperties.js

```diff
@@ -5,13 +5,15 @@
     props["database"] = attr[connectionHelper.attributeDatabase];
     props["dialect"] = "mongosql";
 
-    if (attr["v-log-directory"] == null) {
-        props["LogDir"] = "console";
-    } else {
+    props["LogDir"] = "console";
+    if (attr["v-log-directory"] != null && !attr["v-log-directory"] === "") {
```

Comment:
I tried out the new taco file and this does not work. I always end up with OFF and Console even when changing the LogLevel to FINER for example.
This works : `if (attr["v-log-directory"] != null && attr["v-log-directory"] !== "")`

Reply:
Thanks for checking.  I made the suggested change.